src/Middleware/AuthTokenMiddleware.php

```diff
@@ -99,7 +105,20 @@ public function __invoke(callable $handler)
                 return $handler($request, $options);
             }
 
-            $request = $request->withHeader('authorization', 'Bearer ' . $this->fetchToken());
```

Comment:
Since we're removing the call to `fetchToken`, we should remove the `fetchToken` method below (because it's no longer being used)

Reply:
It's used in line 111

Reply:
Thanks, I missed that!

We could potentially remove it, since we'll have the `tokenCallback` logic in one place, and it's only called once. It could just be done inline like this:

```php
$token = (array) $this->fetcher->fetchAuthToken($this->httpHandler);
$bearer = $token['access_token'] ?? $token['id_token'] ?? '';
$request = $request->withHeader('authorization', 'Bearer ' . $bearer);
```

But that's up to you